hooks/hook-example.py

```diff
@@ -64,6 +64,23 @@ def get_tasks(secret):
 #   ...
 # ]
 #
+# The custom code in compute_labels should loop and label each one of them and 
```

Comment:
@mdipierro can you clarify the input payload?
Doesn't it too have a dictionary of thumbnails by timestamp?